budgetportal/settings.py

```diff
@@ -201,27 +202,26 @@
 # Templates
 TEMPLATES = [
     {
-        'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            str(ROOT_DIR.path("_includes"))
-        ],
-        'APP_DIRS': True,
-        'OPTIONS': {
-            'context_processors': [
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [str(ROOT_DIR.path("_includes"))],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
                 "budgetportal.context_processors.google_analytics",
                 "django.template.context_processors.request",
-            ],
+            ]
         },
-    },
+    }
 ]
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.7/howto/static-files/
 STATICFILES_DIRS = [
     str(PROJ_DIR.path("static")),
-    str(ROOT_DIR.path("assets"))
+    str(ROOT_DIR.path("assets")),
+    str(ROOT_DIR.path("packages/webapp/build/static/js")),
```

Comment:
This is the only real change relevant in this file, the rest is just python black